src/types/BaseField.ts

```diff
@@ -18,25 +23,22 @@ export enum BaseFieldScope {
 
 interface BaseField {
 	readonly id: number;
-	label: string;
-	description: string;
 	shortCode: string;
 	dataType: BaseFieldDataType;
 	scope: BaseFieldScope;
+	readonly localizations: BaseFieldLocalization[];
 	readonly createdAt: string;
 }
 
 type WritableBaseField = Writable<BaseField>;
 
+type WritableBaseFieldWithLocalizations = WritableBaseField & {
+	localizations: WritableBaseFieldLocalizationWithBaseFieldContext[];
+};
+
 const writableBaseFieldSchema: JSONSchemaType<WritableBaseField> = {
 	type: 'object',
 	properties: {
-		label: {
-			type: 'string',
```

Comment:
It's worth calling out that this is a breaking change to the API.

Do we have clients in the wild that will care about this? Changing base fields currently requires admin permissions, so probably we only have internal clients - if any.

I think this is fine, but wanted to double check. @slifty ?

Reply:
True this!  We should be sure to bump the API version (and trigger a new SDK build!)